### PR TITLE
feat(Templates): add creation form uri to template type

### PR DESCRIPTION
**Pull request by ryancarville**
Adds the `creationFromUri` type to the `Templates` type so that it is manageable via the `web-app` and accessible in the `TemplateBlock`.

### CODE REVIEW

File: .changeset/hip-gorillas-explode.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/app-bridge": minor
+---
+
+adds creationFormUri type to template type
```
Comment (ragi96) [inline .changeset/hip-gorillas-explode.md:5]:
```suggestion
feat(AppBridgeBlock): adds `creationFormUri` to the type `template`
```

File: packages/app-bridge/src/tests/TemplateDummy.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export class TemplateDummy {
             name: 'A template',
             description: 'A description',
             previewUrl: 'https://preview.url',
+            creationFormUri: '/publishing/template/1?referer=test.frontify.com',
```
[the remark this answers is not in the document]
Comment (ryancarville) [reply packages/app-bridge/src/tests/TemplateDummy.ts:13]: Good call

**Comment by ragi96** [comment]
Disabled auto-merge as the web-app PR should be ready so we are not blocking others 😉

File: packages/app-bridge/src/tests/TemplateDummy.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ export class TemplateDummy {
             name: 'A template',
             description: 'A description',
             previewUrl: 'https://preview.url',
+            creationFormUri: `/publishing/template/${String(id)}?referer=test.frontify.com`,
```
Comment (ragi96) [inline packages/app-bridge/src/tests/TemplateDummy.ts:13]:
```suggestion
            creationFormUri: `/publishing/template/${id}?referer=test.frontify.com`,
```

Sorry stupid me, you don't need to explicitly convert it, should work like that

**Review by ragi96 — APPROVED** [review]

**Review by henser — APPROVED** [review]
LGTM 👍

**Comment by ragi96** [comment]
Waits for this PR: https://github.com/Frontify/web-app/pull/3733 ready

**Pull request merged**